lib/utils/ConfigParser.js

```diff
@@ -272,6 +256,44 @@ class ConfigParser {
         return this._capabilities
     }
 
+    /**
+     * sets config attribute with file paths from filtering
+     * options from cli argument
+     *
+     * @param  {String} filterObj  list of files in a string from
+     * cli argument
+     * @return {Boolean} exclude  true or false
+     */
+    setFilePathToFilterOptions (filterObj, exclude = false) {
+        const FILTER_TYPE = exclude ? 'exclude' : 'specs'
+
+        const objs = new Set()
+        const objList = filterObj.split(/,/g)
+        const objsList = ConfigParser.getFilePaths(this._config[FILTER_TYPE])
+
+        for (let obj of objList) {
+            if (fs.existsSync(obj) && fs.lstatSync(obj).isFile()) {
+                objs.add(path.resolve(process.cwd(), obj))
+                continue
+            }
+            objsList.forEach(file => {
+                if (file.match(obj)) {
+                    objs.add(file)
+                }
+            })
+        }
+
+        if (objs.size === 0) {
```

Comment:
I don't think we need this anymore given the changed structure of the method.

Reply:
it is still possible to get into this function with a non-existing file hence this condition is still necessary.